### DISCUSSION TITLE
Binance Websocketstream agg@Trade

### DISCUSSION THREAD

**Discussion by AD4910** [community/community#135943] [Programming Help]
### Select Topic Area

Question

### Body

Hello everyone,

I want to monitor the @aggTrade using Websockets for a 15-minute interval. During these 15 minutes, the buys and/or sells should be accumulated. After 15 minutes, an automatic reset should occur, and the calculation starts over again.
My problem is: If the WebSocket connection is interrupted (Timeout, sent 1011 (internal error) keepalive ping timeout; no close frame getting), the accumulated data will be lost, and I won't have the current state anymore.

Does anyone have a solution for this problem?

**Comment by ebndev** [comment]
Hi @AD4910, thanks for posting in GitHub Discussions! 

I've gone ahead and moved this post to our `Programming Help` category, as this topic may be more relevant there and might help someone give you a nudge in the right direction. 😁
